tox.ini

```diff
@@ -1,16 +1,11 @@
 [tox]
-envlist = py{27,36}, coverage, flake8
+envlist = py{27,36}, flake8
 
 [testenv]
-passenv = DISPLAY
-deps = -rrequirements/tests.txt
-commands = pytest --driver Firefox {posargs}
-
-[testenv:coverage]
 passenv = DISPLAY TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 deps = -rrequirements/coverage.txt
 commands =
-    pytest --driver Firefox --cov {posargs}
+    pytest --driver=Firefox --cov --html=results/{envname}.html {posargs}
```

Comment:
Yeah, I saw there were two.. I wonder if it's because of how coveralls uses environment variables from Travis. Unfortunately there's no documentation on this as far as I could find. It might be worth experimenting with.